src/sdk/namespace_cache.js

```diff
@@ -119,6 +120,10 @@ class NamespaceCache {
         return params.object_md.size <= cache_config.DEFAULT_BLOCK_SIZE;
     }
 
+    _update_metric(update_metric_fn_name, value) {
+        this.stats_collector[update_metric_fn_name]({ namespace_resource_id: this.namespace_hub.namespace_resource_id }, value);
+    }
```

Comment:
This will not work, as the `endpoint_report` / `stats_collector` does not expose individual metric setters.
You can see these changes here: https://github.com/noobaa/noobaa-core/pull/6175

Reply:
Ah my bad, I based my changed on @paulxuzhang 's first version